sidebars.js

```diff
@@ -123,155 +168,130 @@ module.exports = {
               items: [
-                'guides/connecting_to_your_data/advanced/database_credentials'
+                'guides/expectations/contributing/how-to-contribute-a-new-expectation-to-great-expectations'
               ]
-            }
-          ]
-        },
-        {
-          type: 'category',
-          label: 'Creating and editing Expectations for your data',
-          items: [
-            { type: 'doc', id: 'guides/expectations/how-to-create-a-new-expectation-suite-using-the-cli' },
-            { type: 'doc', id: 'guides/expectations/how-to-create-a-new-expectation-suite-without-a-sample-batch' },
-            { type: 'doc', id: 'guides/expectations/how-to-create-a-new-expectation-suite-without-the-cli' },
-            { type: 'doc', id: 'guides/expectations/how-to-edit-an-expectation-suite-with-the-cli' },
-            { type: 'doc', id: 'guides/expectations/how-to-edit-an-expectation-suite-without-the-cli' },
-            { type: 'doc', id: 'guides/expectations/how-to-edit-an-expectation-suite-without-a-sample-batch' },
-            { type: 'doc', id: 'guides/expectations/how-to-create-an-expectation-suite-with-the-user-configurable-profiler' },
-            { type: 'doc', id: 'guides/expectations/how-to-create-a-new-expectation-suite-from-a-jsonschema-file' },
-            { type: 'doc', id: 'guides/expectations/how-to-create-custom-expectations' },
-            { type: 'doc', id: 'guides/expectations/how-to-create-custom-expectations-for-pandas' },
-            { type: 'doc', id: 'guides/expectations/how-to-create-custom-expectations-for-spark' },
-            { type: 'doc', id: 'guides/expectations/how-to-create-custom-expectations-for-sqlalchemy' },
-            { type: 'doc', id: 'guides/expectations/how-to-create-expectations-that-span-multiple-batches-using-evaluation-parameters' },
-            { type: 'doc', id: 'guides/expectations/how-to-contribute-a-new-expectation-to-great-expectations' },
-            { type: 'doc', id: 'guides/expectations/how-to-create-parameterized-expectations-super-fast' },
-            { type: 'doc', id: 'guides/expectations/how-to-dynamically-load-evaluation-parameters-from-a-database' }
+            },
+            {
+              type: 'category',
+              label: 'Creating Custom Expectations',
+              items: [
+                'guides/expectations/creating_custom_expectations/how-to-create-custom-expectations',
+                'guides/expectations/creating_custom_expectations/how-to-create-custom-expectations-from-a-sql-query',
+                'guides/expectations/creating_custom_expectations/how-to-create-custom-parameterized-expectations',
+              ]
+            },
           ]
         },
         {
           type: 'category',
           label: '✅ Validating your data',
           items: [
-
             {
               type: 'category',
-              label: 'Working with Validators (name TBD)',
+              label: 'Advanced',
```

Comment:
Missing core skills. Order does not match spec: https://github.com/superconductive/design/blob/main/docs/20210421_knoxpod_golden_paths_mental_map/toc.md#icon-validating-your-data